examples/Thermostat/README.md

```diff
@@ -0,0 +1,87 @@
+# Thermostat
+
+This larger example presents an application with a client (think app running on your phone) that interacts with a device
```

Comment:
```suggestion
This larger example presents an application with a client (ex. an app running on your phone) that interacts with a device
```

I think saying think three times in a row feels a bit repetitive. Breaking this up with ex. might be good.